setup.ts

```diff
@@ -499,6 +532,24 @@ async function main(): Promise<void> {
     await twoFactorAuth();
   }
 
+  if (process.env.LAST_RESORT_SUPERADMIN_EMAIL) {
+    console.log(
+      `\nSuper User email already exists with the value ${process.env.LAST_RESORT_SUPERADMIN_EMAIL}`
```

Comment:
1. We don't have super users, we have super admins.
Specifically in this case it is a super admin of last resort. 1. Please be consistent in the terminologies used in all prompts. It will confuse users following the installation documentation

Reply:
done